_sass/_classes.scss

```diff
@@ -29,8 +29,12 @@
 
 .class-content-container {
     display: grid;
-    grid-template-columns: 30% 70%;
+    grid-template-columns: 230px auto 70%;      //230px is the width of the desktop sidebar +10 px for padding since the ai-content card has no horizontal padding
     min-height: 100vh;
+    @media (max-width: $desktop-width + $width-when-desktop-gets-crowded) and (min-width: $desktop-width) {
+        grid-template-columns: 230px auto;      
+        grid-template-rows: $average-ai-content-card-height auto;     //400px is approximately the height of the ai-content-card
```

Comment:
What happens if you don't set the height - can we make it stretch all the way to the longest card on the row? I'm not particularly worried if each card is not the same height, as long as each card in the same row has the same height.

Reply:
I removed line 36, and it doesn't really affect the content layout by much. So i think i'll remove it.